Formula/syncthing.rb

```diff
@@ -17,15 +17,11 @@ class Syncthing < Formula
   depends_on :hg => :build
 
   def install
-    ENV["GOPATH"] = cached_download/".gopath"
+    ENV["GOPATH"] = buildpath/".syncthing-gopath"
+    mkdir_p buildpath/".syncthing-gopath/src/github.com/syncthing"
+    cp_r cached_download, buildpath/".syncthing-gopath/src/github.com/syncthing/syncthing"
```

Comment:
@mikemcquaid as of syncthing 0.13.0 the build doesn't work with symlinks any longer, that's why it all broke down and the process had to be rewritten. [Here](https://github.com/syncthing/syncthing/issues/3106) is the relevant upstream discussion.